setup/templates/tomcat10/tomcat.conf.template

```diff
@@ -0,0 +1,48 @@
+# OneVizion: from Tomcat 9 package
+# System-wide configuration file for tomcat services
+# This will be loaded by systemd as an environment file,
+# so please keep the syntax. For shell expansion support
+# place your custom files as /etc/tomcat/conf.d/*.conf
+#
+# There are 2 "classes" of startup behavior in this package.
+# The old one, the default service named tomcat.service.
+# The new named instances are called tomcat@instance.service.
+#
+# Use this file to change default values for all services.
+# Change the service specific ones to affect only one service.
+# For tomcat.service it's /etc/sysconfig/tomcat, for
+# tomcat@instance it's /etc/sysconfig/tomcat@instance.
+
+# This variable is used to figure out if config is loaded or not.
+TOMCAT_CFG_LOADED="1"
+
+# Where your java installation lives
+JAVA_HOME="${JAVA_HOME}"
+
+# Where your tomcat installation lives
+CATALINA_HOME="${TOMCAT_PATH}"
+
+# System-wide tmp
+CATALINA_TMPDIR="${TOMCAT_PATH}/temp"
+
+# You can pass some parameters to java here if you wish to
+#JAVA_OPTS="-Xminf0.1 -Xmaxf0.3"
+
+# Use JAVA_OPTS to set java.library.path for libtcnative.so
+#JAVA_OPTS="-Djava.library.path=/usr/lib"
+
+# Set default javax.sql.DataSource factory to apache commons one. See rhbz#1214381
+JAVA_OPTS="-Djavax.sql.DataSource.Factory=org.apache.tomcat.dbcp.dbcp2.BasicDataSourceFactory"
+
+# You can change your tomcat locale here
+#LANG="en_US"
+
+# Run tomcat under the Java Security Manager
+SECURITY_MANAGER="false"
```

Comment:
Why?

Reply:
This is Tomcat 9 default setting (yum package). I tried to up EC2 instance and setup web server using depl-scripts(master). Now I see that java process have no special argument to enable security manager.

```
[root@ip-172-31-13-60 conf]# xargs -0 < /proc/6090/cmdline
/usr/lib/jvm/jre/bin/java -Djavax.sql.DataSource.Factory=org.apache.commons.dbcp.BasicDataSourceFactory -server -Djava.security.egd=file:///dev/urandom -Djava.net.preferIPv4Stack=true -Djava.net.preferIPv4Addresses=true -XX:ErrorFile=/usr/share/tomcat/logs/hs_err_pid%p.log -XX:MetaspaceSize=384m -XX:MaxMetaspaceSize=512m -XX:CompressedClassSpaceSize=128m -Xms2304m -Xmx2304m -XX:+UseG1GC -XX:G1HeapRegionSize=8m -XX:MaxGCPauseMillis=200 -XX:+ParallelRefProcEnabled -XX:+DisableExplicitGC -classpath /usr/share/tomcat/bin/bootstrap.jar:/usr/share/tomcat/bin/tomcat-juli.jar:/usr/share/java/commons-daemon.jar -Dcatalina.base=/usr/share/tomcat -Dcatalina.home=/usr/share/tomcat -Djava.endorsed.dirs= -Djava.io.tmpdir=/var/cache/tomcat/temp -Djava.util.logging.config.file=/usr/share/tomcat/conf/logging.properties -Djava.util.logging.manager=org.apache.juli.ClassLoaderLogManager org.apache.catalina.startup.Bootstrap start
```

![image](https://github.com/user-attachments/assets/53884419-3198-419a-b4db-7071e2083238)

Reply:
Asked in Teams PM

Reply:
I dropped this file because of official package. Official package has SECURITY_MANAGER="false"